neofetch

```diff
@@ -11389,7 +11412,6 @@ oMMMMmMMMMNds:.+MMMmmMMN/.-odNMMMMmMMMM+
              .odNm+  /dNms.
 EOF
                 ;;
-
```

Comment:
I meant remove it from the diff, not the file.

Reply:
It's unrelated to this PR.

Reply:
Sorry, I am confused